LibraryApp/src/app/AppContainer.js

```diff
@@ -28,7 +31,8 @@ const BooksNavigator = createStackNavigator(
       headerTitleStyle: styles.headerTitleStyles,
       headerBackTitle: null,
       headerTransparent: true,
-      headerBackImage: <Image source={backImg} style={styles.headerBackImgStyles} />
+      headerBackImage: <Image source={backImg} style={styles.headerBackImgStyles} />,
+      headerRight: <CustomButton title={LOGOUT_TEXT} buttonStyles={styles.logoutButtonStyle} />
```

Comment:
This doesnt work right now, do this?

Reply:
at this point it is only a decorative button 👀